utilities.py

```diff
@@ -8,30 +14,41 @@
 
 
 def parameter_file_load(x):
-    try:
-        with open(x) as json_file:
-            text_input = pd.read_json(json_file)
-            return text_input
-    except IOError:
-        print("File Handling Error: ", IOError)
+    if os.path.exists(x):
+        try:
+            with open(x) as json_file:
+                text_input = pd.read_json(json_file)
+                return text_input
+        except Exception as e:
+            logging.error("There was issue to Open\Read the file Due to: " + str(e))
+    else:
+        logging.error("Could not find the query file in " + os.path.dirname(x))
 
 
 def oracle_connect(db_ip, db_user, db_pass, db_sid):
     try:
         dsn_tns = orcl.makedsn(host=db_ip, port=1521, service_name=db_sid)
         return orcl.connect(user=db_user, password=db_pass, dsn=dsn_tns)
-    except orcl.DatabaseError:
-        raise
+    except Exception as e:
+        logging.error("There was issue with DB connection Due to: " + str(e))
+        pass
 
 
 def read_query(x):

```

Comment:
rename x to more meaning full argument name

Reply:
done